tasks/plugins.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Install plugins
+  shell: "sudo -u www-data php occ app:install -n {{ item }}"
+  args:
+    chdir: "{{ nextcloud_path }}"
+  with_items: "{{ nextcloud_plugins }}"
+  register: result
+  failed_when: result.rc != 0 and not "already installed" in result.stdout
+  changed_when: not "already installed" in result.stdout
+
+- name: Activate plugins
+  shell: "sudo -u www-data php occ app:enable -n {{ item }}"
+  args:
+    chdir: "{{ nextcloud_path }}"
+  with_items: "{{ nextcloud_plugins }}"
+  register: result
```

Comment:
`result` is not used.
Also, why not change this task?

Reply:
yes it is see when condition on update nextcloud

Reply:
No this is another variable that is coincidentally also named `result`.

Reply:
fixed